backend/api/communication/src/main/java/co/airy/core/api/communication/SendMessageController.java

```diff
@@ -64,7 +64,8 @@
                 .setHeaders(Map.of())
                 .setDeliveryState(DeliveryState.PENDING)
                 .setSource(channel.getSource())
-                .setSenderId(auth.getPrincipal().toString())
+                // TODO add the oidc id here in https://github.com/airyhq/airy/issues/1518
+                .setSenderId("airy-core-anonymous")
```

Comment:
@airyhq/codeowners-backend Even with oidc we will have to keep a default sender id for when there is no authentication configured